crates/sui-types/src/committee.rs

```diff
@@ -284,6 +285,24 @@ impl Committee {
             .binary_search_by_key(name, |(a, _)| *a)
             .is_ok()
     }
+
+    // ===== Testing-only methods =====
+
+    /// Generate a simple committee with 4 validators each with equal voting stake of 1.
+    pub fn new_simple_test_committee() -> (Self, Vec<AuthorityKeyPair>) {
```

Comment:
`#[cfg(test)]`?

Reply:
We cannot put #[cfg(test)] because we expect it to be used outside of sui-types crate